packages/sdk/src/interfaces/llm.ts

```diff
@@ -28,7 +31,12 @@ const MessageSchema = z.object({
       z.array(
         z.object({
           type: z.enum(['text', 'image']),
-          mimeType: z.string(),
+          mimeType: z
```

Comment:
**chore:** same thing here. A string becoming optional in an action input is non-breaking, but in an action output, it is. I believe this schema is used both in the input and output, so this is a breaking change that will require bumping the integration and interface version (like the one above)

Reply:
Done in https://github.com/botpress/botpress/pull/13159/commits/424a05c6a7d843c9ce3c46fa1cbde0be22e51ec3